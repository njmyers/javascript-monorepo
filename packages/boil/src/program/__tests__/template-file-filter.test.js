import templateFileFilter from '../template-file-filter';

describe('it works', () => {
  test('it is a function ', () => {
    expect(typeof templateFileFilter()).toBe('function');
  });

  test('it runs without crashing', () => {
    expect(templateFileFilter()).toMatchSnapshot();
  });
});
