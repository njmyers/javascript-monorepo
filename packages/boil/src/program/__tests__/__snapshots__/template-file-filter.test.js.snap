// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`program/templateFileFilter it runs without crashing 1`] = `true`;
